### DISCUSSION TITLE
у меня не работает микрофон хотя я даю разрешение на использование этой хуйни просто опять просит разрешение

### DISCUSSION THREAD

**Discussion by xlaas** [stoatchat/discussions#831] [Q&A]
у меня не работает микрофон хотя я даю разрешение на использование этой хуйни просто опять просит разрешение

**Comment by thewoolfi** [comment]
Это визуальный баг, а так все работает. Присоединяешься, даёшь разрешение и говоришь.

WOAH (RU)

**Comment by PivBoyarin** [comment]
та же проблема, даю разрешение на использование микрофона, закрываю окно, захожу в голосовой чат и не могу включить микрофон, 
заново открываю настройки и вижу, что требуется выдать разрешение

**Reply by VasiliyKos** [reply]
+1
